### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by witold-swierzy**

### CODE REVIEW

**Comment by witold-swierzy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by AlexanderHodicke** [comment]
Next time, please add your Manager as a reviewer

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/README.md
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/README.md:1]: can you please rename the folder to lowercase with - example:  oracle-jdbc-23c-new-features

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/pom.xml:1]: @oheimburger code review please

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/jdb23cnfcdemotable.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
+drop table if exists test;
```
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/jdb23cnfcdemotable.sql:1]: @oheimburger code review please

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/DMLSubscriber.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package swierzy.oraclejdbc23newfeatures;
```
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/DMLSubscriber.java:1]: @oheimburger code review, please

File name and folder maybe strange.

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/Main.java [hunk 4]
```diff
@@ -0,0 +1,156 @@
+package swierzy.oraclejdbc23newfeatures;
```
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/Main.java:1]: @oheimburger code review please

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/QuerySubscriber.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
+package swierzy.oraclejdbc23newfeatures;
```
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/QuerySubscriber.java:1]: @oheimburger code review please

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]

**Review by JesusGitHubOracle — COMMENTED** [review]
Hi Alex - noted.

File: data-platform/core-converged-db/OracleJDBC23cNewFeatures/README.md
Comment (oheimburger) [reply data-platform/core-converged-db/OracleJDBC23cNewFeatures/README.md:1]: This looks like a duplicate of PR 53.
I would reject this.

**Review by oheimburger — CHANGES_REQUESTED** [review]
Looks like a duplicate of PR 53.

**Pull request closed without merging**